src/main/java/frc/robot/commands/TeleopClawActuationCommand.java

```diff
@@ -0,0 +1,30 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.lib.DTXboxController;
+import frc.robot.subsystems.grabber.GrabberClaw;
+
+public class TeleopClawActuationCommand extends CommandBase{
+    private DTXboxController controller;
+    private GrabberClaw claw;
+
+    @Override
+    public void execute(){
+        boolean aButton = controller.getAButtonPressed();
+        //close for cone ^
+        boolean bButton = controller.getBButtonPressed();
+        //close for cube ^
+        boolean yButton = controller.getYButtonPressed();
```

Comment:
This is how we would write the teleopPeriodic method in a TimedRobot. For a command-based bot, we need 3 commands to open, close (for cube) and close (for claw). Then we bind them to the buttons in the RobotContainer. We can reuse the same commands for autonomous.

Reply:
Completed in recent push!